regression/pages/whitelabel/const.py

```diff
@@ -0,0 +1,335 @@
+        'course-v1:MITProfessionalX+MITPX04+2016': 200.0,
+        'course-v1:MITProfessionalX+MITPX05+2016': 300.0,
+        'course-v1:MITProfessionalX+MITPX06+2016': 300.0,
+    },
+    'MITProfessionalX': {
+        'course-v1:HarvardXPLUS+HXP01+2016 ': 100.0,
+        'course-v1:HarvardXPLUS+HXP02+2016 ': 100.0,
+        'course-v1:HarvardXPLUS+HXP03+2016 ': 200.0,
+        'course-v1:HarvardXPLUS+HXP04+2016 ': 200.0,
+        'course-v1:HarvardXPLUS+HXP05+2016 ': 300.0,
+        'course-v1:HarvardXPLUS+HXP06+2016 ': 300.0
+    }
+}
+
+##############################################################################
+
+# REGISTRATION INFORMATION
+REG_INFO = {
+    'full_name': 'White label Test User',
+    'first_name': 'White Label',
+    'last_name': 'Test User',
+    'gender': 'm',
+    'yob': '1994',
+    'state': 'Massachusetts',
+    'country': 'US',
+    'edu_level': 'm',
+    'company': 'Arbisoft',
+    'title': 'SQA'
+}
+
+# BILLING INFORMATION
+BILLING_INFO = {
+    'first_name': 'billing',
+    'last_name': 'user',
+    'address01': '23-b',
+    'address02': 'service lane',
+    'city': 'Boston',
+    'country': 'US',
+    'state': 'MA',
+    'postal_code': '02108',
+    'email': 'billing_user@example.com'
+}
+
+# PAYMENT DETAILS
+PAYMENT_DETAILS = {
+    'card_number': '4111111111111111',
+    'cvn': '123',
+    'expiry_month': '07',
+    'expiry_year': '2018'
+}
+
+# STAGES IN PAYMENT SECTION
+INITIAL_STAGE = 'Review'
+FINAL_STAGE = 'Confirmation'
+
+
+# Existing user email
+EXISTING_USER_EMAIL = 'wl_smoke_user01@example.com'
+
+# Staff user email
+STAFF_EMAIL = os.environ['STAFF_USER_EMAIL']
+
+# Student user email
+VISUAL_USER_EMAIL = 'wl_visual_test01@example.com'
+
+# Countries list
+COUNTRIES = [
+    u'Afghanistan', u'\xc5land Islands', u'Albania', u'Algeria',
+    u'American Samoa', u'Andorra', u'Angola',
+    u'Anguilla', u'Antarctica', u'Antigua and Barbuda', u'Argentina',
+    u'Armenia', u'Aruba', u'Australia',
+    u'Austria', u'Azerbaijan', u'Bahamas', u'Bahrain', u'Bangladesh',
+    u'Barbados', u'Belarus', u'Belgium',
+    u'Belize', u'Benin', u'Bermuda', u'Bhutan', u'Bolivia',
+    u'Bonaire, Sint Eustatius and Saba',
+    u'Bosnia and Herzegovina', u'Botswana', u'Bouvet Island', u'Brazil',
+    u'British Indian Ocean Territory',
+    u'Brunei', u'Bulgaria', u'Burkina Faso', u'Burundi', u'Cabo Verde',
+    u'Cambodia', u'Cameroon', u'Canada',
+    u'Cayman Islands', u'Central African Republic', u'Chad', u'Chile',
+    u'China', u'Christmas Island',
+    u'Cocos (Keeling) Islands', u'Colombia', u'Comoros', u'Congo',
+    u'Congo (the Democratic Republic of the)',
+    u'Cook Islands', u'Costa Rica', u"C\xf4te d'Ivoire", u'Croatia', u'Cuba',
+    u'Cura\xe7ao', u'Cyprus',
+    u'Czech Republic', u'Denmark', u'Djibouti', u'Dominica',
+    u'Dominican Republic', u'Ecuador', u'Egypt',
+    u'El Salvador', u'Equatorial Guinea', u'Eritrea', u'Estonia',
+    u'Ethiopia', u'Falkland Islands [Malvinas]',
```

Comment:
I think installing a library is much better than a list that you're going to have to maintain and fix over time. (Unless you're attempting to test the library itself, but since it's a 3rd-party library, those kinds of tests should exist on that library.) It is more DRY to re-use that library rather than constructing our own list. It's true that we'll be installing more into the project, but for the most part an installed library is not a big deal.

Taking a step back though, why do you need all the countries?